syntax = "proto2";
package spider;

message pingpong_resp {
	required int32 num = 1;
}
